dpp/src/util/json_schema.rs

```diff
@@ -87,34 +88,25 @@ impl JsonSchemaExt for JsonValue {
 
     fn get_indices(&self) -> Result<Vec<Index>, anyhow::Error> {
         match self.get("indices") {
-            Some(raw_indices) => Ok(serde_json::from_value(raw_indices.to_owned())?),
+            Some(raw_indices) => {
+                let indices_with_raw_properties: Vec<IndexWithRawProperties> =
```

Comment:
Nor sure about usage. But I guess we have new RS Drive primitives to get this info.